app/views/shared/_header_nav.haml

```diff
@@ -32,25 +32,11 @@
         - if has_access_to_test_results_index?
           %li{:class => ["test_results", "encounters"].include?(params[:controller]) && "active"}
             = link_to "Tests", test_results_path
-        - if has_access_to_sites_index?
-          %li{:class => params[:controller] == "sites" && "active"}
-            = link_to "Sites", sites_path
-        - if has_access_to_devices_index?
-          %li{:class => params[:controller] == "devices" && "active"}
-            = link_to "Devices", devices_path
-        - if has_access?(Device, Policy::Actions::SUPPORT_DEVICE)
-          %li{:class => params[:controller] == "device_messages" && "active"}
-            = link_to "Logs", device_messages_path
-        - if has_access_to_device_models_index?
-          %li{:class => params[:controller] == "device_models" && "active"}
-            = link_to "Models", device_models_path
-        - if has_access_to_users_index?
-          -# Devise clears the controller param for users index
-          %li{:class => (params[:controller] == "users" || !params[:controller]) && "active"}
-            = link_to "Users", users_path
-        - if has_access_to_settings?
-          %li{:class => (["roles", "policies", "alerts", "incidents", "alert_messages"].include?(params[:controller]) || params[:action] == "settings") && "active"}
-            = link_to "Settings", settings_path
+
+  - if has_access_to_settings?
+    = link_to settings_path do
+      .icon-gray-light.medium{title: "Settings"}
+        = '<svg xmlns="http://www.w3.org/2000/svg" style="width: 0.9em;" aria-hidden="true" viewBox="0 0 24 24"><path fill="currentColor" d="M12 15.5A3.5 3.5 0 0 1 8.5 12A3.5 3.5 0 0 1 12 8.5a3.5 3.5 0 0 1 3.5 3.5a3.5 3.5 0 0 1-3.5 3.5m7.43-2.53c.04-.32.07-.64.07-.97c0-.33-.03-.66-.07-1l2.11-1.63c.19-.15.24-.42.12-.64l-2-3.46c-.12-.22-.39-.31-.61-.22l-2.49 1c-.52-.39-1.06-.73-1.69-.98l-.37-2.65A.506.506 0 0 0 14 2h-4c-.25 0-.46.18-.5.42l-.37 2.65c-.63.25-1.17.59-1.69.98l-2.49-1c-.22-.09-.49 0-.61.22l-2 3.46c-.13.22-.07.49.12.64L4.57 11c-.04.34-.07.67-.07 1c0 .33.03.65.07.97l-2.11 1.66c-.19.15-.25.42-.12.64l2 3.46c.12.22.39.3.61.22l2.49-1.01c.52.4 1.06.74 1.69.99l.37 2.65c.04.24.25.42.5.42h4c.25 0 .46-.18.5-.42l.37-2.65c.63-.26 1.17-.59 1.69-.99l2.49 1.01c.22.08.49 0 .61-.22l2-3.46c.12-.22.07-.49-.12-.64l-2.11-1.66Z"/></svg>'.html_safe
```

Comment:
I'm not sure how the process for adding icons to the font works. I just did this quick and dirty for now.

Reply:
I documented the process quickly [app/assets/fonts/README.md](https://github.com/instedd/cdx/blob/main/app/assets/fonts/README.md).

I reproduced it last week to add the Boxes icon.